Tallies/TallyClerks/mgXsClerk_class.f90

```diff
@@ -228,13 +231,10 @@ subroutine reportInColl(self, p, xsData, mem, virtual)
     character(100), parameter :: Here =' reportInColl (mgXsClerk_class.f90)'
 
     ! Return if collision is virtual but virtual collision handling is off
-    if (self % virtual) then
-      ! Retrieve tracking cross section from cache
-      flux = p % w / xsData % getTrackingXS(p, p % matIdx(), TRACKING_XS)
-    else
-      if (virtual) return
-      flux = p % w / xsData % getTotalMatXS(p, p % matIdx())
-    end if
+    if ((.not. self % virtual) .and. virtual) return
+
+    ! Ensure we're not in void (could happen when scoring virtual collisions)
+    if (p % matIdx() == VOID_MAT) return
```

Comment:
I added that check to avoid trying to look up cross sections in void. Now, for example, I removed this check from the collisionClerk (so that one is able to calculate the flux in void) but I added it to macroResponse to avoid look ups in void. 

I guess in the mgXsClerk I could score a flux in void but set all the other xss to zero, that would make it correct. I can have a look at that in this PR, I don't see why postponing!